.github/workflows/sns_lifecycle.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         docker kill $SNS_TESTING_INSTANCE
 
   publish:
-    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    if: ${{ github.event_name != 'pull_request' }}
```

Comment:
Wouldn't it be simpler to simply remove the subexpression that uses "main"?

Reply:
I.e. keep 'push', but just do not restrict based on ref.

Rationale: `!= 'whatever'` encompasses a many things that are not explicitly mentioned.